### ISSUE TITLE
While configuring roadrunner-dep-llvm-3.5.2 produces error Unexpected failure executing llvm-build

### ISSUE THREAD

**Issue by debashish05** [sys-bio/roadrunner-dep-llvm-3.5.2#1]
**Error**
While configuring llvm dependency for libroadrunner in cmake 3.14.3 I am facing an error 
CMake Error at CMakeLists.txt:375 (message):
  Unexpected failure executing llvm-build: llvm-build: fatal error: missing
  LLVMBuild.txt file at:
  'D:\\buildingroadrunner\\source\\roadrunner-dep-llvm-3.5.2/projects\\LLVMBuild.txt'

I am using the Windows operating system. It is due to missing file LLVMBuild.txt. 

**Possible Patch:**
I downloaded llvm source code from http://releases.llvm.org/download.html and just moved the LLVMBuild.txt text file from Project folder to the llvm dependency it worked fine. It only contains 3 lines.
[LLVMBuild.txt](https://github.com/sys-bio/roadrunner-dep-llvm-3.5.2/files/3531312/LLVMBuild.txt)

**Comment by andrewbriand** [comment]
Interesting, I also get the same error trying to build from source on Ubuntu. I found [this](https://groups.google.com/forum/#!topic/appleseed-dev/CE3uVH3cPR4) thread which discusses a similar issue and proposes the same fix as you, so perhaps that will work fine. Let me know if you're able to build successfully that way.

**Comment by debashish05** [comment]
Yeah. I am able to build it successfully. While working with ubuntu I used the official llvm 3.5.2 it works too.
